src/angular-local-storage.js

```diff
@@ -302,7 +302,10 @@ angularLocalStorage.provider('localStorageService', function() {
           expiryDate.setTime(expiryDate.getTime() + (-1 * 24 * 60 * 60 * 1000));
           expiry = "; expires=" + expiryDate.toGMTString();
           value = '';
-        } else if (isNumber(daysToExpiry) && daysToExpiry !== 0) {
+          //if the value is undefined set the cookie to session cookie
+        } else if (daysToExpiry == undefined) {
```

Comment:
but in this case i need to have a Session cookie for handle Oauth 2.0, so any other solution with this lib?